src/simplification/Simplifier.ts

```diff
@@ -76,6 +85,15 @@ export default class Simplifier {
           model.enum = enums;
         }
       }
+      if(this.options.allowInheritance){
+        const simplifiedExtends = simplifyExtend(schema, this);
+        if(simplifiedExtends.newModels !== undefined){
```

Comment:
if `newModels` has signature `{ newModels: CommonModel[] }` why we check undefined case? 😄

Reply:
It does not? `CommonModel[] | undefined` 
I use undefined instead of checking length of array, since that makes more sense 😄

Reply:
ahh, sorry... I did not notice it 😄